examples/pna-example-tcp-connection-state-tracking-simplified.p4

```diff
@@ -0,0 +1,249 @@
+Copyright 2022 AMD
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <core.p4>
+#include "../pna.p4"
+
+
+// Sample PNA program demonstrating the use of write back table
+// entries. Although a possible syntax is proposed to indicate
+// action data that is written back to a table, the purpose of
+// this example is not to support that specific syntax, but
+// rather the usefulness and advantages of being able to write
+// action data back to a table to store state information
+
+// NOTE: this example uses a simplified version of the code in
+// pna-example-tcp-connection-tracking.p4 to add entries
+// in the TCP connection table (ct_tcp_table).
+// Although the same naming is purposedly used for the connection
+// table, the removal of entries (due to time out or explicit connection
+// tear-down is explicitly not supported for the sake of keeping the code
+// simpler and more legible.
+// A complete implementation of TCP connection and status tracking
+// would require to include such functionalities that can be found in
+//  pna-example-tcp-connection-tracking.p4
+
+
+// NOTE: the code is currently written for a compiler supporting
```

Comment:
no need for this comment. if/else are allowed in action routine by the language.

Reply:
There was a whole big recent discussion on whether add_entry_if should be added to PNA or not, and if so, why, related to some P4 targets not supporting if statements within action bodies.  I hope you are right that P4 implementations for NICs that do not support if within actions will be rare, but they might still exist for a while.